### ISSUE TITLE
Error installing on version 2.4.0b under win64

### ISSUE THREAD

**Issue by mobius1qwe** [LongDirtyAnimAlf/fpcupdeluxe#648]
![image](https://github.com/LongDirtyAnimAlf/fpcupdeluxe/assets/26689802/e6601c9b-7a43-41ed-a107-8d8962b1a2db)
trying to update lazarus and/or FPC to latest version using the "Stable" button or the "Install/Update FPC+Lazarus" button gives me this error of the picture.
I'm on Win10 Pro 21H2 x64

22/12/2023 16:31:44: fpcupdeluxe: V4821 (20231222) started.
FPCUPdeluxe V2.4.0b for x86_64-win64 running on Win64-10.0.19045
Build with: FPC 3.2.2 on Win11 x86_64

**Comment by gcarreno** [comment]
Hey Mobius(@mobius1qwe),

This is what happens when you try to update any tag.
In the days of `SVN`, there was an option to allow this.
I don't think this works now that we have `GIT`.

I stumbled into this same issue when I tried to update stable from whatever came before 2.2.6 into 2.2.6.
I remember that I tried going to options and turned the thing on, but I was unsuccessful.

I'm now creating a new folder for each of the new stable versions.

I know, I know, it's a pain, but I'm not really sure if we are able to get around it.
But since I'm not a `GIT` expert, I'll defer to them on the subject, in case I'm missing something here.

Grande abraço,
Gus
